lib/CommandDeclarations.ts

```diff
@@ -88,6 +89,14 @@ import { handleWeather } from './Weather.js';
 import { config } from './Config.js';
 
 export const Commands: Command[] = [
+    {
+        aliases: ['burnt'],
+        primaryCommand: {
+        argsFormat: Args.Combined,
+        implementation: handleBurnt,
+        description: 'Displays the current number of slugs burnt by the incinerator',
```

Comment:
```suggestion
            argsFormat: Args.Combined,
            implementation: handleBurnt,
            description: 'Displays the current number of slugs burnt by the incinerator',
```
these should be indented

Reply:
Args aren't used, can change to `Args.DontUse` and remove from handleBurnt signature